{{cookiecutter.app_name}}/README.rst

```diff
@@ -135,3 +135,56 @@ should cache all your assets forever by including the following line
 in your ``settings.py``::
 
     SEND_FILE_MAX_AGE_DEFAULT = 31556926  # one year
+
```

Comment:
Nit: Could these extra newlines be added inside of the `if` block? They will only be necessary if the deployment instructions are added.

Reply:
Oh, you are right :)